### PR TITLE
Ks7

### PR DESCRIPTION
**Pull request by santiaago**
ks7 shows the changes needed to have an in memory todo list app
will be the ground work for ks8 and forward where we introduce databases

### CODE REVIEW

File: ks7/app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        'Content-Type': 'application/json'
+      }
+    }).then(response => {
+      return response.json()
+    }).then(json => {
+      this.setState({ message: json.message })
+    })
+  }
+
+  onTaskAdded(taksName) {
```
Comment (cjheppell) [inline ks7/app/src/App.js]: minor: typo in param name. `taksName` instead of `taskName`

File: ks7/app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    }).then(response => {
+      return response.json()
+    }).then(json => {
+      this.setState({ todoItems: json.todoList})
+    })
+  }
+
+  onTaskUpdate(item){
+    const itemToUpdate = this.state.todoItems.find(x => x.id === item.id)
+    itemToUpdate.done = !item.done
```
Comment (cjheppell) [inline ks7/app/src/App.js]: Isn't this modifying the react `state` object? I think we need to do an `Object.assign()` call instead to make a copy of the object and update the copy.

File: ks7/server/server.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+'ks6 web server'
```
Comment (cjheppell) [inline ks7/server/server.py]: minor: this should be ks7

File: ks7/app/src/App.js (hunk 1 above)
Comment (santiaago) [reply ks7/app/src/App.js]: 👍

File: ks7/app/src/App.js (hunk 2 above)
Comment (santiaago) [reply ks7/app/src/App.js]: nice catch 👍

File: ks7/server/server.py (hunk 3 above)
Comment (santiaago) [reply ks7/server/server.py]: 👍

**Pull request merged**